src/app/zap-templates/zcl/data-model/chip/application-launcher-cluster.xml

```diff
@@ -80,5 +80,8 @@ limitations under the License.
     <item name="Success" value="0x00"/>
     <item name="AppNotAvailable" value="0x01"/>
     <item name="SystemBusy" value="0x02"/>
+    <item name="PendingUserApproval" value="0x03"/>
+    <item name="Downloading" value="0x04"/>
+    <item name="Installing" value="0x05"/>
```

Comment:
I don't see these in the spec anywhere.  Where is this coming from?

Reply:
@bzbarsky-apple Chris (@chrisdecenzo) will file a CCB next week July 9th.

Reply:
The right order, generally, is to either change the spec or at least have a spec PR up, then point to that changed spec or pending PR in the SDK PR.  Otherwise it's absolutely impossible to review the SDK change.

Reply:
Makes sense. Here's the spec PR https://github.com/CHIP-Specifications/connectedhomeip-spec/pull/10014